ansible/roles/nexus-container/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+## Sonatype Nexus Install
+
+##
+  - name: Download Nexus template to master
+    get_url:
+      url: https://raw.githubusercontent.com/OpenShiftDemos/nexus/master/nexus{{nexus_version}}-persistent-template.yaml
+      dest: /root/nexus-template.yaml
+      mode: 0440
+      
+  - name: Check if Nexus was already provisioned
+    command: "oc get service nexus -n {{desired_project}}"
+    register: install_nexus
+    ignore_errors: true
```

Comment:
can you add `changed_when: false`, as this is read-only ?

Reply:
👍